### PR TITLE
small improvements and rename `name/names` methods

### PR DESCRIPTION
**Pull request by siegy22**
In my opinion the `names` method should be called `text`, because it's just a simple text, which has nothing to do with a name.

You can squash the commits into one to merge them :grin:

### CODE REVIEW

**Comment by siegy22** [comment]
Gonna fix this later, failed because of the alias_method.

File: lib/inputs.rb [hunk 1]
```diff
@@ -8,30 +8,23 @@ def self.yn(question)
...
       input = input.chomp
     end until %w(y n).include?(input.chomp)
-
-    case input
-    when 'y'
-      true
-    when 'n'
-      false
-    end
+    input == 'y'
```
Comment (equivalent) [inline lib/inputs.rb:11]: this is welcomed

File: lib/inputs.rb [hunk 2]
```diff
@@ -8,30 +8,23 @@ def self.yn(question)
...
   end
 
-  def self.name(question)
+  def self.text(question)
```
Comment (equivalent) [inline lib/inputs.rb:14]: ...but please don't change this in PR , feel free to keep it in your fork but wont be able to accept this (due to backwards compatibility)

File: lib/inputs.rb [hunk 3]
```diff
@@ -8,30 +8,23 @@ def self.yn(question)
...
-    name = _input_evaluator.call
-    name = name.chomp
-    output name.green
-    name
+    _input_evaluator.call.chomp
   end
+  alias_method :name, :text
 
-  def self.names(question)
+  def self.multiple_text(question)
```
Comment (equivalent) [inline lib/inputs.rb:20]: ...but please don't change this in PR , feel free to keep it in your fork but wont be able to accept this  (due to backwards compatibility)

File: README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ The gem aims to be really simple. If you looking for something more complex I re
 
 [Reddit discussion](https://www.reddit.com/r/ruby/comments/4i5dep/gem_inputs_another_pointless_gem_for_handling/)
 
-```
+```ruby
```
Comment (equivalent) [inline README.md:12]: :+1:

**Review by equivalent — CHANGES_REQUESTED** [review]

File: lib/inputs.rb (hunk 2 above)
Comment (aimaard) [reply lib/inputs.rb:14]: it's still backwards compatible if you alias it with the old name..

**Pull request closed without merging**